File: PHP/AVAPHP/tvAVA.php
<!-- Classe TV: Faça um programa que simule um televisor criando-o como um objeto. O usuário deve ser capaz de informar o número do canal e aumentar ou diminuir o volume. Certifique-se de que o número do canal e o nível do volume permanecem dentro de faixas válidas. -->

<?php
    class Televisor {
        private $canalAtual = 0;
        private $listaCanal = [];
        private $volume = 5;
        public function adicionarCanal ($canalNovo){
            if (in_array($canalNovo, $listaCanal)){
                echo "esse canal ja foi adicionado";
            }else{
                $this -> listaCanal = array_push($listaCanal, $canalNovo);
            }
        }
        public function mudaCanal ($canal){
            if (in_array($canalNovo, $listaCanal)){
                $this -> listaCanal = $canal;
            }else{
                echo "Esse canal n foi adicionado";
            }
        }
        public function aumentaVolume(){
            if ($volume += 1 > 10 ){
                echo "Volume maxmo";
            }else{
                $this -> volume += 1;
            }
        }
        public function diminuirVollume(){
            if ($volume -= 1 < 0 ){
                echo "Volume minimo";
            }else{
                $this -> volume -= 1;
            }
        }
    }
?>

File: Python/tvAVA.py
# Classe TV: Faça um programa que simule um televisor criando-o como um objeto. O usuário deve ser capaz de informar o número do canal e aumentar ou diminuir o volume. Certifique-se de que o número do canal e o nível do volume permanecem dentro de faixas válidas.
class Televisao :
    def __init__(self):
        self.__canalAtual = 0
        self.__listaCanais = []
        self.__volume = 5
    def adicionarCanal (self,canalNovo):
        if canalNovo not in self.__listaCanais:
            self.__listaCanais = self.__listaCanais.append[canalNovo]
            print(f'canal {canalNovo} adicionado')
        else :
            print('este canal ja foi adicionado ')
    def mudarCanal (self, canal):
        if canal not in self.__listaCanais :
            print('canal nao adicionado')
        else :
            self.__canalAtual = canal
    def aumentarVolume (self):
        if self.__volume + 1 > 10 :
            print('Volume maximo')
        else :
            self.__volume += 1
            print(f'Volume {self.__volume}')
    def diminuirVolume (self):
        if self.__volume - 1  < 0 :
            print('Volume minimo ')
        else :
            self.__volume -= 1
            print(f'Volume {self.__volume}')
